app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
> The bad side of using github.io is that we depend on GitHub.

I don't think that that is a problem, we depend on GitHub anyway, and I think when something happens, then the schemes is our last problem. And a user of diaspora shouldn't even notice it, if there is a problem with the schemes on GitHub.

> What do we do if the gem is updated but schema isn't?

We can just change the schema version when it changes (to the gem version with which the new schema is released), and if the schema doesn't change, keep it on the last version where it changed. That would result in versions with gaps, but I don't think that is a problem.

If you really think it is worth to track it's own version for the schemes then I think it's the best to extract it to it's own project and add it as dependency.

But anyway: I'm interested in @denschub opinion on this topic.